src/Core/Hook/HookDispatcher.php

```diff
@@ -77,7 +78,7 @@ public function dispatchRendering(HookInterface $hook)
             $hook->getParameters()
         );
 
-        $content = $event->getContent();
+        $content = $event instanceof RenderingHookEvent ? $event->getContent() : [];
```

Comment:
I'm not 100% sure this is not changing the behavior of the HookDispatcher

Reply:
Fixed. The PhpDoc in `Adapter/Hook/HookDispatcher.php` was updated at an another moment.